dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -686,7 +686,7 @@ private void setHTMLPagesDependencies(Set<String> idsToWork) {
 						structuresSet.add(containerStructure.getStructureId());
 					}
 
-					List<MultiTree> treeList = MultiTreeFactory.getMultiTree(workingPage,container);
+					List<MultiTree> treeList = MultiTreeFactory.getMultiTrees(workingPage,container);
 
 					for (MultiTree mt : treeList) {
 						String contentIdentifier = mt.getChild();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getChild"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)